### ISSUE TITLE
Vulnerability roundup 101: file-roller-3.36.3: 1 advisory [3.9]

### ISSUE THREAD

**Issue by ckauhaus** [NixOS/nixpkgs#120373]
[search](https://search.nix.gsc.io/?q=file-roller&i=fosho&repos=NixOS-nixpkgs), [files](https://github.com/NixOS/nixpkgs/search?utf8=%E2%9C%93&q=file-roller+in%3Apath&type=Code)

* [ ] [CVE-2020-36314](https://nvd.nist.gov/vuln/detail/CVE-2020-36314) CVSSv3=3.9 (nixos-20.09)

Scanned versions: nixos-20.09: c7e905b6a97.

Cc @dasj19
Cc @hedning
Cc @jtojnar
Cc @worldofpeace

**Comment by dasj19** [comment]
Could be fixed by 3.36.4 : https://download.gnome.org/sources/file-roller/3.36/

**Comment by ckauhaus** [comment]
The CVE states that the bug is present in versions up to 3.38 (including), but this information could be out of date. Confirm anyone?

**Comment by avdv** [comment]
Seems to be indeed fixed by 3.36.4, commit https://gitlab.gnome.org/GNOME/file-roller/-/commit/4aaa7c65140762d5cf0df335a7bb795f242d18a8

Basically, the CVE was assigned around the same time the fix was implemented and awaits reanalysis.

**Comment by avdv** [comment]
What's the next step here? Since the PR https://github.com/NixOS/nixpkgs/pull/124471 is merged, should we close this ticket; or only after it gets merged into nixos-20.09 proper?

**Comment by ckauhaus** [comment]
The ticket should be closed after the fix has hit the street, which means nixos-20.09 in this case. I see that it now has passed Hydra, so we can complete this.
